core/modules/system/system.install

```diff
@@ -2814,6 +2827,13 @@ function system_update_1056() {
 }
 
 /**
+ * Change default state for display error messages.
+ */
+function system_update_1057() {
+  config_set('system.core', 'error_level', 'hide');
```

Comment:
Hm. This isn't changing the default state, this is changing everyone's actual setting messaging.  I'd be tempted to remove the update since there are lots of development sites out there today that might want this setting to stay where it is. @quicksketch what do you think?